src/lib/mina_base/transaction_logic.ml

```diff
@@ -1295,14 +1292,6 @@ module Make (L : Ledger_intf) : S with type ledger := L.t = struct
           ~error:Update_not_permitted_delegate
       else return a.delegate
     in
-    let snapp =
-      let snapp_version =
-        (* Current snapp version. Upgrade mechanism should live here. *)
-        Mina_numbers.Snapp_version.zero
```

Comment:
You've removed this. Do we need to add a field in Party.t for updating `snapp_version`?